export function throwError(msg: string): void {
  throw new Error(msg)
}